metricbeat/module/docker/docker.go

```diff
@@ -38,6 +38,7 @@ import (
 // Select Docker API version
 const dockerAPIVersion = "1.22"
 
+// HostParser is a TCP host parser function for docker tcp host addresses
 var HostParser = parse.URLHostParserBuilder{DefaultScheme: "tcp"}.Build()
```

Comment:
I guess it's a common pattern in metricbeat. Instead of making it a public variable, how about exporting a function and making the parser a private variable?